lang/po/new/cs.po

```diff
@@ -1293,7 +1289,7 @@ msgstr "Zvuk"
 # MSG_RUNOUTS c=7
 #: ultralcd.cpp:1593
 msgid "Runouts"
-msgstr ""
+msgstr "Runouts"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.